maxim/blocks/grid_gating.py

```diff
@@ -66,7 +63,7 @@ def apply(x):
         )(y)
         y = layers.Dropout(dropout_rate)(y)
         x = x + y
-        x = UnblockImages()(x, grid_size=(gh, gw), patch_size=(fh, fw))
+        x = TFUnblockImages()(x, grid_size=(gh, gw), patch_size=(ph, pw))
```

Comment:
Same. You're changing the semanticity of the code. Could you please elaborate why?

Reading this change and also previous `x, ph, pw = TFBlockImagesByGrid()(x, grid_size=(gh, gw))` and comparing them to their previous versions -- they don't read the same too.